pybamm/input/parameters/lithium-ion/anodes/graphite_Chen2020_plating/parameters.csv

```diff
@@ -0,0 +1,36 @@
+Name [units],Value,Reference,Notes
+# Empty rows and rows starting with ‘#’ will be ignored,,,
+,,,
+# Electrode properties,,,
+Negative electrode conductivity [S.m-1],215,Chen 2020,graphite
+Maximum concentration in negative electrode [mol.m-3],33133,Chen 2020,tuned for 1C
+Negative electrode diffusivity [m2.s-1],[function]graphite_LGM50_diffusivity_Chen2020,Chen 2020,tuned for 1C
+Negative electrode OCP [V],[data]graphite_LGM50_ocp_Chen2020,Chen 2020,
+,,,
+# Microstructure,,,
+Negative electrode porosity,0.25,Chen 2020,
+Negative electrode active material volume fraction,0.75,Chen 2020,
+Negative particle radius [m],5.86E-6,Chen 2020,
+Negative electrode Bruggeman coefficient (electrolyte),1.5,Chen 2020,theoretical
+Negative electrode Bruggeman coefficient (electrode),1.5,default,
+,,,
+# Interfacial reactions,,,
+Negative electrode cation signed stoichiometry,-1,,
+Negative electrode electrons in reaction,1,,
+Reference OCP vs SHE in the negative electrode [V],,,
+Negative electrode charge transfer coefficient,0.5,Chen 2020,
+Negative electrode double-layer capacity [F.m-2],0.2,,
+Negative electrode exchange-current density [A.m-2],[function]graphite_LGM50_electrolyte_exchange_current_density_Chen2020,,
+,,,
+# Density,,,
+Negative electrode density [kg.m-3],1657,default,
+,,,
+# Thermal parameters,,,
+Negative electrode specific heat capacity [J.kg-1.K-1],700,default,
+Negative electrode thermal conductivity [W.m-1.K-1],1.7,default,
+Negative electrode OCP entropic change [V.K-1],0,,
+,,,
+# Li plating parameters,,,
+Li metal partial molar volume [m3.mol-1],1.3E-5,,
```

Comment:
For the plating parameters, add the reference where you got them from in the third column (like in the Negative electrode charge transfer coefficient, for example).